src/mono/sample/iOS/Makefile

```diff
@@ -10,6 +10,8 @@ DEPLOY_AND_RUN?=true
 APP_SANDBOX?=false
 STRIP_DEBUG_SYMBOLS?=false # only used when measuring SOD via build-appbundle make target
 HYBRID_GLOBALIZATION?=true
+USE_MONO_RUNTIME?=true
+AOT?=true
```

Comment:
Does this addition of AOT mean that we were not actually measuring AOT when we thought we were?

Reply:
Does CoreCLR look at `AOT` variable? I would expect that does nothing with CoreCLR since it has `ReadyToRun`.